canvas_modules/common-canvas/__tests__/common-properties/components/title-editor-test.js

```diff
@@ -94,172 +109,184 @@ describe("title-editor renders correctly", () => {
 				help={help}
 			/>
 		);
-		const helpButton = wrapper.find(".properties-title-editor-btn[data-id='help']").hostNodes();
-		expect(helpButton).to.have.length(1);
-		helpButton.simulate("click");
+		const { container } = wrapper;
+		const helpButtonCont = container.querySelector(".properties-title-editor-btn[data-id='help']");
+		expect(helpButtonCont).to.exist;
```

Comment:
here can you add assertion to check if there is only 1 help button ?

Reply:
I added it